backend/middleware/sanitize-input.js

```diff
@@ -0,0 +1,25 @@
+//Author: Cleveland Plonsey
+//Purpose: defines a middleware function that sanitizes user input
+//Date: 11/9/2023
+
+const { validationResult, body } = require("express-validator");
+
+//sanitize the input from the user. This prevents Cross-site scripting (XSS)
+//and SQL injection attacks
+//add this middleware to any route that accepts user input and interacts with the db
```

Comment:
I would use docstring for this rather than multiple single line comments if you want to keep this